assets/ca/ca.querytranslator.js

```diff
@@ -255,6 +271,19 @@ var caUI = caUI || {};
 	};
 
 	/**
+	 * Determine whether the given list of tokens is all words and whitespace.
+	 * @param {Array} tokens
+	 * @returns {boolean}
+	 */
+	isRawSearchText = function (tokens) {
```

Comment:
I don't really like the implementation of this function but without `lodash` it's good enough.